app/bundles/LeadBundle/Controller/LeadController.php

```diff
@@ -1609,6 +1609,12 @@ public function emailAction($objectId = 0)
                             );
                         } else {
                             $errors = $mailer->getErrors();
+
+                            if (isset($errors['failures'])) {
+                                $errors[] = $this->get('translator')->trans('mautic.lead.for.email', array('%emails%' => implode('<br />', $errors['failures'])));
```

Comment:
I'm not sure the extra error message is needed since we are sending the mail to a single lead/email and thus can see the email on the lead's profile or in the list?  I think I inadvertently fixed this in #1211 but I'll remove my fix in favor of this that addresses the bug directly.